src/main/java/interface_adapter/delete_clothing_item/DeleteViewModel.java

```diff
@@ -0,0 +1,36 @@
+package interface_adapter.delete_clothing_item;
+
+import interface_adapter.ViewModel;
+import java.beans.PropertyChangeListener;
+import java.beans.PropertyChangeSupport;
+
+public class DeleteViewModel extends ViewModel {
+    public static final String TITLE_LABEL = "Delete Clothing Item";
+    public static final String DELETE_BUTTON_LABEL = "Delete";
+
```

Comment:
Cancel button?

Reply:
I am not sure if we are going to have a cancel button in our UI for Delete Clothing Item use case.